app/src/main/res/values/ids.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What is this file for? I don't see why we need it.

Reply:
I don't clearly remember.  It may be system generated?  It might be being used in MainActivity.java line 71 for example.  It's been a while since I did this/Android coding.  We can try without it and see if stuff breaks. :man_shrugging:

Reply:
I just tested it and it works without it. To save the trouble of going back and forth on the pull request for things like this, I'm going to reject the PR and just make your changes on my branch and push them up.